.github/CODEOWNERS

```diff
@@ -0,0 +1,81 @@
+# Describes which individuals/teams are reponsible for which code in the
+# repository.
+#
+# NOTE: ownership by an individual is a smell, as is ownership by multiple
+# teams. We have a soft goal of arranging our code so that each entry names
+# a single team as owner. Better not to hide the truth, though; we'd rather
+# be honest about where we still have individuals as owners.
+#
+# See: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/.cargo                             @benesch
+/.config                            @benesch
+/.devcontainer                      @mjibson
+/.github                            @benesch
+/bin
+/ci                                 @MaterializeInc/qa
+/doc/user                           @MaterializeInc/DevEx
+/doc/developer/reference/compute    @MaterializeInc/compute
+/doc/developer/reference/storage    @MaterializeInc/storage
+/src/adapter                        @MaterializeInc/surfaces
+/src/alloc                          @benesch
+/src/audit-log                      @MaterializeInc/surfaces
+/src/avro                           @umanwizard
+/src/avro-derive                    @umanwizard
+/src/build-id                       @umanwizard
+/src/build-info                     @benesch
+/src/ccsr                           @benesch
+/src/cloud-resources                @MaterializeInc/cloud
+/src/clusterd                       @MaterializeInc/compute @MaterializeInc/storage
```

Comment:
we say it's a smell, but this one will probably always have multiple teams?

Reply:
There is a world in which we introduce a "timely dataflow team" that is distinct from either the storage or compute team!